UC-Berkeley-Extension/global
# Remove duplicate Landing Page Content Type

## Description
Wanted to clean up our content types before starting to build a database of record. 

Also ran composer update.

## Testing Instructions
1. `phing install`
1. Verify that only one Landing Page content type is listed and that it includes Layout Builder.
